### PR TITLE
Number of chars

### PR DESCRIPTION
**Pull request by CryptoSFor**

### CODE REVIEW

**Comment by xartishuk** [comment]
`readme.md` should have info inside. `.gitignore` should have stuff inside too

**Comment by xartishuk** [comment]
rename the repo, "task1" won't do. Same goes for the package

**Comment by xartishuk** [comment]
commit messages should be descriptive. Here's a link from general requirements: https://chris.beams.io/posts/git-commit/

**Comment by xartishuk** [comment]
if you want to version your code, you should take a look at git tags

**Review by xartishuk — CHANGES_REQUESTED** [review]
see comments

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This project contains two functions "NumberOfCharacters" and "ReverseString", the first you can use to count characters in a string, the second to reverse the string. Also, both functions were covered with unit tests.
```
Comment (xartishuk) [inline README.md]: you can use 
>`



symbol to denote code, like so: 
`NumberOfCharacters`

File: task1.go [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package str
+
+func NumberOfCharacters(s string) int {
```
Comment (xartishuk) [inline task1.go]: add a comment

**Review by xartishuk — CHANGES_REQUESTED** [review]

**Comment by xartishuk** [comment]
+ rename the file from task1.go to smth else

File: README.md (hunk 1 above)
Comment (CryptoSFor) [reply README.md]: Done

File: task1.go (hunk 2 above)
Comment (CryptoSFor) [reply task1.go]: Done

Comment (xartishuk) [reply task1.go]: is it? don't see it

Comment (CryptoSFor) [reply task1.go]: Done!

**Comment by xartishuk** [comment]
comments look good. The rename `str.go -> number_of_chars.go` doesn't look good. Try to avoid long file names. `str.go, string.go, strings.go, char.go, charnum.go, char-num.go, char_num.go, charNum.go` are all fine

**Review by xartishuk — APPROVED** [review]

**Pull request merged**